sdk/storage/inc/datalake/path_client.hpp

```diff
@@ -207,18 +226,71 @@ namespace Azure { namespace Storage { namespace DataLake {
     /**
      * @brief Sets the properties of a resource the path points to.
      * @param options Optional parameters to set the properties to the resource the path points to.
-     * @return PathUpdateResponse
+     * @return SetPropertiesResult
      */
-    PathUpdateResponse SetProperties(
+    SetPropertiesResult SetProperties(
         const SetPathPropertiesOptions& options = SetPathPropertiesOptions()) const;
 
+    /**
+     * @brief Get Properties returns all system and user defined properties for a path. Get Status
+     *        returns all system defined properties for a path. Get Access Control List returns the
+     *        access control list for a path.
+     * @param options Optional parameters to get the properties from the resource the path points
+     *                to.
+     * @return GetPathPropertiesResponse
+     */
+    GetPathPropertiesResponse GetProperties(
+        const PathGetPropertiesOptions& options = PathGetPropertiesOptions()) const;
+
+    /**
+     * @brief Sets the metadata of a resource the path points to.
+     * @param options Optional parameters to set the metadata to the resource the path points to.
+     * @return SetMetadataResult
+     */
+    SetMetadataResult SetMetadata(
```

Comment:
Sometimes this is `XxxResponse`, while sometimes this is `XxxResult`.

Reply:
Changed all return value to either be a `XxxResponse` or something like `FileSystemProperties`.